server/nc-sa-2018-013.json

```diff
@@ -0,0 +1,40 @@
+{
+   "Title": "Session fixation on public share page",
+   "Timestamp": 1540468800,
+   "Risk": 1,
+   "CVSS3": {
+     "score": 3.1,
+     "vector": "AV:N/AC:H/PR:H/UI:R/S:U/C:L/I:L/A:N"
+   },
+   "CWE": {
+      "id": 384,
+      "name": "Session Fixation"
+   },
+   "HackerOne": 237184,
+   "Affected":[
+      {
+         "Version":"14.0.0",
+         "CVE":"CVE assignment pending",
+         "Operator":"<"
+      },
+      {
+         "Version":"13.0.3",
+         "CVE":"CVE assignment pending",
+         "Operator":"<"
+      },
+      {
+         "Version":"12.0.8",
+         "CVE":"CVE assignment pending",
+         "Operator":"<"
+      }
+   ],
+   "Description":"<p>A bug causing session fixation could potentially allow an attacker to obtain access to password protected shares.</p>",
+   "ActionTaken": "<p>The error has been fixed.</p>",
+   "Acknowledgment":[
+      {
+         "Name": "Anonymous hacker",
+         "Reason": "Vulnerability discovery and disclosure."
+      }
+   ],
+   "Resolution": "<p>It is recommended that all instances are upgraded to at least Nextcloud 14.0.0, Nextcloud 13.0.3 or Nextcloud 12.0.8.</p>"
```

Comment:
Numbers are off, those are the affected versions.

Reply:
Nope affected is `<`